README.rst

```diff
@@ -161,3 +171,4 @@ Common description. For a detailed description, use ``tt help command`` .
 * ``completion`` - generate autocomplete for a specified shell.
 * ``help`` - display help for any command.
 * ``logrotate`` - rotate logs of a started tarantool instance.
+* ``rocks`` - LuaRocks package manager.
```

Comment:
This command does not exist, it is created in the next commit. I suggest that you insert the README as the last commit in the patchset.

Reply:
fixed